Docs/source/developers/profiling.rst

```diff
@@ -208,3 +212,80 @@ In these examples, the individual lines for recording a trace profile are:
 Now open the created trace files (per rank) in the Nsight-Systems GUI.
 This can be done on another system than the one that recorded the traces.
 For example, if you record on a cluster and open the analysis GUI on your laptop, it is recommended to make sure that versions of Nsight-Systems match on the remote and local system.
+
+Nvidia Nsight-Compute
+---------------------
+
+`Vendor homepage <https://developer.nvidia.com/nsight-compute>`__ and `product manual <https://docs.nvidia.com/nsight-compute/>`__.
+
+Nsight-Compute captures fine grained information at the kernel level
+concerning resource utilization. It collects a lot of data, and runs slowly (can
```

Comment:
Just FYI: Slowlyness depends on the stats and number of stats to collect.
What makes it slow is that for complicated stats, it has to run multiple times the same kernel. To achieve that, it does swap the whole GPU memory in and out. Wild, isn't it.

Reply:
Very nifty, and I guess the cache state doesn't matter cause they'll have to flush it anyways.
I added this to the note in the docs. Maybe it'll help people who are trying to run more steps.